model/deployment.go

```diff
@@ -139,6 +140,9 @@ type Deployment struct {
 	// Status is the overall deployment status
 	Status DeploymentStatus `json:"status" bson:"status"`
 
+	// Active is true for unfinished deployments
+	Active bool `json:"-" bson:"active"`
```

Comment:
just out of curiosity: is there a benefit of using this over relying on the absence of the [finished timestamp](https://github.com/mendersoftware/deployments/blob/master/model/deployment.go#L126)?

Reply:
We need a partial index (filter by X, sort by Y). The documentation is here:
https://docs.mongodb.com/manual/core/index-partial/#create-a-partial-index

Unfortunately, "$not" is not supported. It means you can create a partial index for "$exists", but not for the negation of this clause. In our case, we want to get the active deployments (pending/inprogress). Therefore, we need the `{"$not": { "$exists": true}}`.

Reply:
Good to know, thank you!